src/main/java/com/google/maps/model/AddressType.java

```diff
@@ -234,6 +234,10 @@ public String toString() {
     return addressType;
   }
 
+  public String toCanonicalLiteral() {
```

Comment:
That's a reasonable request, now that I understand the context. Can you please add tests to cover the two `toCanonicalLiteral` calls you have added, just to make sure we don't break these while moving forwward. Thanks!